CHANGELOG.md

```diff
@@ -64,6 +64,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Added `Ipv6DontFrag` for android, iOS, linux and macOS.
 - Added `IpDontFrag` for iOS, macOS.
   (#[1692](https://github.com/nix-rust/nix/pull/1692))
+- Added `ptrace::read_user` and `ptrace::write_user` for Linux.
```

Comment:
Can you rebase such that this is in the latest unreleased section (the current changes it's near are part of the `0.24.0` release) and also add a link back to this PR, like the other items in the CHANGELOG?

Reply:
Done. Thanks!